src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ReactElement, useMemo } from 'react'
+import { Modal as AntModal } from 'antd'
+import classNames from 'classnames'
+
+import { Body } from './Modal.Body'
+import { Footer } from './Modal.Footer'
+import { ModalProps } from './Modal.props'
+import { Size } from './Modal.types'
+import { Title } from './Modal.Title'
+import styles from './Modal.module.css'
+
+const { Small, Large, FullScreen } = Size
+const {
+  modal,
+  modalSm,
+  modalLg,
+  modalFs,
+} = styles
+
+/**
+ * Modal dialog that opens in overlay and allows to perform flows of actions, or to show specific
+ * but less relevant information than those displayed on the underlying page.
+ *
+ * @link https://ant.design/components/modal
+ * @returns {Modal} Modal component
+ */
+export const Modal = ({
+  aside,
+  children,
+  docLink,
+  footer,
+  isBodyFullWidth,
+  isClosable,
+  isVisible,
+  onCloseClick,
+  size,
+  title,
+}: ModalProps): ReactElement => {
+  const modalClassNames = useMemo(() => classNames([
+    modal,
+    size === Small && modalSm,
+    size === Large && modalLg,
+    size === FullScreen && modalFs,
+  ]), [size])
+
+  return (
+    <AntModal
+      centered
+      className={modalClassNames}
+      closable={isClosable}
+      footer={<Modal.Footer footer={footer} />}
+      keyboard={isClosable}
+      maskClosable={false}
```

Comment:
I'd map this prop to the isClosable flag.

Reply:
closable is used to show the X to close modal. 
the prop quoted to you is already mapped below as` closable={isClosable}`

Reply:
yes i know, I'd map `maskClosable` as well, a modal that is not closable by clicking outside of it is extremely annoying.

Reply:
I check this with the designer and @Giogia anche this props needs to be false

Reply:
This design choice makes very little sense to me. Let's keep it this way for now but i'll open an issue for this.